views/error.pug

```diff
@@ -11,13 +11,15 @@ html
     meta(http-equiv="Content-Type", content="text/html; charset=utf-8")
     meta(name="viewport", content="width=device-width, initial-scale=1")
     title #{__('TITLE', community)}
+    link(href="//static.uqcs.org/bulma/main.css" rel="stylesheet" type="text/css")
     link(href="css/style.css", rel="stylesheet", type="text/css")
```

Comment:
Would this 404 since you deleted the file?

Reply:
The style file still exists as it contains some site specific CSS